doc/source/api_changes.rst

```diff
@@ -2,6 +2,14 @@
  Release History
 =================
 
+v1.5.8 (2020-06-05)
+===================
+
+Fixed
+-----
+
+* Missing implementation details of :func:`~bluesky.run_engine.RunBundler.collect` have been added.
+
```

Comment:
Should we mention it's a backported change?

Reply:
good idea